test/gen-server/AuthCaching.ts

```diff
@@ -50,12 +50,17 @@ describe('AuthCaching', function() {
     setUpDB();
     await createInitialDb();
     process.env.GRIST_DATA_DIR = testDocDir;
-    homeServer = await mergedServerMain(0, ['home'],
+
```

Comment:
Sorry reverting the MergedServer change is going to touch so many files, but on the plus side, shorter PR...

Reply:
Closing this because MergedServer is required.